### ISSUE TITLE
Optical flow

### ISSUE THREAD

**Issue by alexarje** [fourMs/MGT-python#81]
It would be very nice to add support for optical flow. This could be done either within the mgmotion module, or, perhaps better, as a separate module. It should be possible to generate a motion video with optical flow arrows, and possibly also use this for creating a motiongram.

**Comment by balintlaczko** [comment]
Hi there! Hope you don't mind the cross-commenting, just to provide some info, I have made a basic implementation if this now, and here is the closing comment of my issue (ie. the "fork" of this issue in my fork of this repo) on this:

Implemented optical flow - in a basic manner for now, which means that all the params of the algorithms are "dumped" on the user, which is OK for people who are computer vision scientists, and quite mysterious for everyone else. But it works, and it can give some meaningful results without specifying anything.
Here is an example usage of the sparse optical flow:
```
from mgmodule import MgObject as Mg
Mg('dance.avi', starttime=5, endtime=10).flow.sparse().show()
```
and the dense optical flow:
```
from mgmodule import MgObject as Mg
Mg('dance.avi', starttime=5, endtime=10).flow.dense().show()
```

(...oh yeah, and chaining is actually working now with other stuff too, check out the jupyter notebook from the fork repo: https://github.com/balintlaczko/MGT-python/blob/master/MotionGesturesToolbox.ipynb )
